.github/workflows/staging-deploy.yml

```diff
@@ -23,10 +28,14 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: staging
-      - name: remove and replace files 
+      - name: modify files
+      # removes .git folder to avoid conflicts with other repo 
+      # inserts value for key baseurl: "/website-staging" 
+      # appends the repository key and value to the end of the _config file
         run: |
           rm -rf .git
-          mv _config_staging.yml _config.yml 
+          sed 's/\(^baseurl:\).*/\1 "\/website-staging" /' _config.yml 
+          'echo >> _config.yml && echo "repository: "usds/website-staging"" >> _config.yml'
```

Comment:
Unless I've missed it, the `repository` key is not used anywhere. I think it can be removed.